test/functionalTest/testHarness.sh

```diff
@@ -25,6 +25,7 @@
 date
 testStartTime=$(date +%s.%2N)
 MAX_TRIES=${CB_MAX_TRIES:-3}
+echo $testStartTime > /tmp/brokerStart
```

Comment:
Yes, I need to remove the file /tmp/brokerStart somewhere, as only '>>' is used.
If I don't do that, the file would grow and grow and grow ...